src/renderer/src/components/settings/SettingsView.tsx

```diff
@@ -0,0 +1,43 @@
+import SettingDropdown from './SettingDropdown';
+import "../../assets/css/settings/settings-view.css";
+import "../../assets/css/settings/settings-item.css";
+import { createEffect, createSignal, onMount } from 'solid-js';
+import { changeAudioDevice} from '../../lib/Music';
+
+const settingsView = () => {
+  let view;
+
+  const [audioDevices, setAudioDevices] = createSignal(new Map<string, ()=>any>());
+
+  const setAudioDevicesMap = () => {
+    const audioMap = new Map<string, ()=>any>();
+    navigator.mediaDevices.enumerateDevices().then(r => {
+      for (const device of r) {
+        if (device.kind === "audiooutput") {
+          audioMap.set(device.label, () => changeAudioDevice(device.deviceId));
+        }
+      }
+      setAudioDevices(audioMap);
+      console.log(audioDevices());
```

Comment:
Shouldn't this be removed?

Reply:
the console log? yeah, I was supposed to remove it